plugins/woocommerce-admin/client/homescreen/activity-panel/orders/utils.js

```diff
@@ -74,7 +70,7 @@ export function getLowStockCount( select ) {
 	// depend on `getItemsTotalCount` to have been called.
 	// eslint-disable-next-line @wordpress/no-unused-vars-before-return
 	const totalLowStockProducts = getItemsTotalCount(
-		'products/low-in-stock',
+		'products/count-low-in-stock',
```

Comment:
We should also update the call on line 83 and 86 below: https://github.com/woocommerce/woocommerce/blob/aea1e858931f389d258184eed61646a6a8140413/plugins/woocommerce-admin/client/homescreen/activity-panel/orders/utils.js#L79-L82

I believe this is causing the infinite loading screen when updating the last stock item.